Makefile

```diff
@@ -7,12 +7,14 @@ SKOPEO_BRANCH = master
 SUDO =
 
 ifeq ($(shell uname),Darwin)
-DARWIN_BUILD_TAG = containers_image_ostree_stub
+	DARWIN_BUILD_TAG = containers_image_ostree_stub
```

Comment:
Isn’t this indentation risky? If a rule somehow ended up above the `ifeq`, this would be interpreted as a shell command.

Reply:
Good catch. Theoretically, you are right. I think the risk for this is happening is quite low, but let's move it back.